patches/tModLoader/Terraria/ModLoader/Core/TmodFile.cs

```diff
@@ -45,6 +46,8 @@ internal FileEntry(string name, int offset, int length, int compressedLength, by
 
 	private static string Sanitize(string path) => path.Replace('\\', '/');
 
+	private static readonly SHA1 sha1 = SHA1.Create();
```

Comment:
Does this actually make a difference to performance? We're not multithreading right now, but it's a reduction in encapsulation and potential for future mistakes.